README.md

```diff
@@ -511,6 +502,151 @@ in: <header>
 | x-exosite-prefix       | string | `token`       | apiKey   | A string to prefix in the header content.<br>E.g. X-secret: token `<user token>`                                        |
 | x-exosite-from         | string | `token`       | apiKey   | **Required**. The parameter name containing the token value to add in the Authentication header or query parameter. |
 
+##### Basic
+
+[Standard Basic Authentication](https://tools.ietf.org/html/rfc7617).
+
+Example:
+```yaml
+type: basic
+x-userField: <account>
+x-secretField: <secret>
```

Comment:
should this be?
```
x-exosite-user-field: <account>
x-exosite-secret-field: <secret>
```

Reply:
yes, I just copied the existing section but it was indeed outdated.. Ill change that.
```suggestion
x-exosite-secret-field: <secret>
```